src/Person.tsx

```diff
@@ -336,7 +335,7 @@ export default function Person(props: Props) {
             <span className="">
               {person.name + " "}
               <i
-                style={{ color: colors[person.id - (1 % colorNames.length)] }}
+                style={{ color: colors[person.id - (1 % colors.length)] }}
```

Comment:
```suggestion
                style={{ color: colors[(person.id - 1) % colors.length] }}
```

This was my mistake earlier. The colors aren't wrapping around properly. 
![colors_mod](https://user-images.githubusercontent.com/2104990/92771285-98d0ad00-f34f-11ea-8c7d-ef7eec1cce8b.png)